build.gradle

```diff
@@ -21,7 +21,7 @@ subprojects {
 }
 
 ext {
-    minSdkVersion = 9
+    minSdkVersion = 14
     compileSdkVersion = 28
-    sourceCompatibility = JavaVersion.VERSION_1_7
+    sourceCompatibility = JavaVersion.VERSION_1_8
 }
```

Comment:
I don't want to change this. The library is effectively done and won't benefit from any language feature of 8 in a meaningful way.